connectors/openai/element-templates/openai-connector.json

```diff
@@ -170,11 +166,16 @@
     },
     {
       "label": "Model",
-      "description": "Select the model",
+      "id": "model",
+      "description": "Select the model (or Custom for free text)",
       "group": "input",
       "type": "Dropdown",
       "value": "gpt-3.5-turbo",
       "choices": [
+        {
+          "name": "Custom",
+          "value": "custom"
+        },
         {
           "name": "GPT-3.5-turbo",
           "value": "gpt-3.5-turbo"
```

Comment:
Would be great if we could add the other known supported ones as well https://platform.openai.com/docs/models/continuous-model-upgrades namely: `gpt-4` and `gpt-4-turbo-preview`

Reply:
Another reference: https://platform.openai.com/docs/models/model-endpoint-compatibility

Reply:
Done ✔️